examples/basic_cog.py

```diff
@@ -0,0 +1,113 @@
+import discord
+from discord.ext import commands
+import random
+
+description = """An example bot to showcase the discord.ext.commands extension
+module.
+There are a number of utility commands being showcased here.
+The example uses cogs to organize the code
+"""
+
+# The intents chosen are the default intents which allow us to get all but presence and member intents.
+# We use the member intent for the `on_member_join` event.
+# For more information on intents read https://discordpy.readthedocs.io/en/latest/intents.html#intents-primer.
+intents = discord.Intents.default()
+intents.members = True
+
+bot = commands.Bot(command_prefix='?', description=description, intents=intents)
+
+
+# This defines a class that contains commands in a 'Miscellaneous' category.
+# The library calls this concept a Cog and they must inherit from `commands.Cog`.
+# Cogs are useful for grouping commands and having them share state.
+# This class will group commands in a 'Misc' category in the default help command.
+# The `name` keyword argument passed allows us to set the name of the Cog.
+# If a name is not given then it will default to the name of the class.
+# Read more here: https://discordpy.readthedocs.io/en/latest/ext/commands/cogs.html.
+
+
+class Miscellaneous(commands.Cog, name='Misc'):
+    """
+    A simple Miscellaneous cog.
+    """
+
+    def __init__(self, bot):
+
+        # We pass in the bot parameter when initializing the class, and make it an instance attribute.
+        # This lets us access the bot parameter throughout our methods.
+        self.bot = bot
+        # You can also add other attributes to the class.
+        self.afk_reasons = {}
+
+    # Cogs also have the capability to define checks that apply to every command in the Cog.
+    # They work similarly to regular checks elsewhere in the framework.
+    # ref: https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html#checks
+    async def cog_check(self, ctx):
+        # Check if the command was invoked within a Guild channel.
+        return ctx.guild is not None
+
+    # Commands within a Cog are defined using the commands.command decorator
+    # instead of the bot.command decorator.
+    # They must also pass a `self` parameter since it's within a class.
+    @commands.command()
+    async def hello(self, ctx):
+        """Have the bot greet you."""
+        await ctx.send('Hello am I an example bot!')
```

Comment:
```suggestion
        await ctx.send('Hello I\'m an example bot!')
```

Reply:
The better change would have been without the apostrophe.